src/components/home/FriendSelectBox/index.tsx

```diff
@@ -1,15 +1,19 @@
 import { useTheme } from "@emotion/react";
 import { FiPlus } from "react-icons/fi";
 import { boxStyle, plusIconStyle, textStyle } from "./styles";
+import { useAuth } from "@/contexts/AuthContext";
 
 export default function FriendSelectBox() {
+  const { username } = useAuth();
   const theme = useTheme();
   return (
     <div css={boxStyle(theme)}>
       <div css={plusIconStyle(theme)}>
         <FiPlus />
       </div>
-      <span css={textStyle(theme)}>선물할 친구를 선택해 주세요.</span>
+      <span css={textStyle(theme)}>
+        {username ? `${username}님! ` : ""}선물할 친구를 선택해 주세요.
```

Comment:
디테일 차이이긴 하지만 저라면 이렇게 할 것 같아요!
```suggestion
        {username ? `${username}님! 선물할 친구를 선택해 주세요.` : "선물할 친구를 선택해 주세요."}
```